README.md

```diff
@@ -2,3 +2,7 @@
 emnify Tech-Radar: a tool to visualize technology choices, inspire and support Engineering teams at emnify to pick the best technologies for new projects
 
 https://emnify.github.io/tech-radar/
+
+## Contribution Guide
+
+To suggest changes please change the `radar.json` file and make a PR.
```

Comment:
I think it'd be good to add the mapping here for the rings and quadrants to avoid mistakes in PRs.